assets/operator/okd-x86_64/ruby/imagestreams/ruby-centos.json

```diff
@@ -78,6 +78,27 @@
 					"type": "Local"
 				}
 			},
+			{
```

Comment:
OK this matches what I see in https://github.com/sclorg/s2i-ruby-container/pull/318/files so that is good